website/cue/reference/remap/functions/parse_grok.cue

```diff
@@ -3,7 +3,7 @@ package metadata
 remap: functions: parse_grok: {
 	category:    "Parse"
 	description: """
-		Parses the `value` using the [`grok` format](\(urls.grok)). All patterns [listed here](\(urls.grok_patterns))
+		Parses the `value` using the [`grok`](https://github.com/daschl/grok/tree/master/patterns) format. All patterns [listed here](\(urls.grok_patterns))
```

Comment:
Ah, yeah, the grok online debugger disappeared. I couldn't find another so dropping this makes sense. This link make sense, but could we update it in the `urls.cue` file instead of here?